ros2topic/ros2topic/api/__init__.py

```diff
@@ -54,6 +54,23 @@ def __call__(self, prefix, parsed_args, **kwargs):
                     parsed_args, self.include_hidden_topics_key))
 
 
+def import_message_type(topic_name, message_type):
+    # TODO(dirk-thomas) this logic should come from a rosidl related package
+    try:
+        package_name, message_name = message_type.split('/', 2)
+        if not package_name or not message_name:
+            raise ValueError()
+    except ValueError:
+        raise RuntimeError('The passed message type is invalid')
+
+    middle_module = 'msg'
+    if topic_name.endswith('/_action/feedback'):
+        middle_module = 'action'
```

Comment:
Please add a `TODO` for this block since this hack needs to be replaced.

Reply:
Added TODO and rebased to keep this as two commits